coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -3351,6 +3351,20 @@ def _internal_op_tensor_inplace_fill(context, node):
     data = context[node.inputs[0]]
     fill_scalar = context[node.inputs[1]]
 
+    if len(node.inputs) == 2 and not any_symbolic(fill_scalar.val):
```

Comment:
A `Var` object's `.val` will never be symbolic:
https://github.com/apple/coremltools/blob/main/coremltools/converters/mil/mil/var.py#L202
I will suggest to check `fill_scalar.val is not None`.
That is more precise.